roles/contiv_network/templates/netplugin.j2

```diff
@@ -0,0 +1,7 @@
+{# If an interface is specified for netplugin, use it #}
+{# Otherwise, hope for the best                       #}
```

Comment:
The existing scripts do not pass this argument. I did not want to break existing stuff. That's why I did not bail out here.